include/functions.h

```diff
@@ -916,7 +916,7 @@ void collision_check_player_overlaps(void);
 void update_player_input(void);
 void phys_update_action_state(void);
 void collision_main_lateral(void);
-void func_800EFD08(void);
+void handle_floor_behaviour(void);
```

Comment:
we use US spelling in the repo (sorry) so these should be "behavior" - also below